src/lib/protocols/aimini.c

```diff
@@ -262,7 +262,7 @@ void init_aimini_dissector(struct ndpi_detection_module_struct *ndpi_struct, u_i
   ndpi_set_bitmask_protocol_detection("Aimini", ndpi_struct, detection_bitmask, *id,
 				      NDPI_PROTOCOL_AIMINI,
 				      ndpi_search_aimini,
-				      NDPI_SELECTION_BITMASK_PROTOCOL_V4_V6_TCP_OR_UDP_WITH_PAYLOAD,
```

Comment:
Do we even need `NDPI_SELECTION_BITMASK_PROTOCOL_V4_V6_TCP_OR_UDP_WITH_PAYLOAD` anymore?

Reply:
Not really; it is better to remove them to avoid "copy-and-paste" errors in new code.
If we need them again, we will re-add in the future.